src/storage/hummock_sdk/src/version.rs

```diff
@@ -579,35 +579,49 @@ where
             .collect()
     }
 
+    pub fn newly_added_object_ids_exclude_change_log(&self) -> HashSet<HummockSstableObjectId> {
+        self.newly_added_sst_infos_exclude_change_log()
+            .map(|sst| sst.object_id())
+            .collect()
+    }
+
     pub fn newly_added_sst_ids(&self) -> HashSet<HummockSstableObjectId> {
         self.newly_added_sst_infos()
             .map(|sst| sst.sst_id())
             .collect()
     }
 
+    pub fn newly_added_sst_ids_exclude_change_log(&self) -> HashSet<HummockSstableObjectId> {
+        self.newly_added_sst_infos_exclude_change_log()
+            .map(|sst| sst.sst_id())
+            .collect()
+    }
+
     pub fn newly_added_sst_infos(&self) -> impl Iterator<Item = &'_ T> {
```

Comment:
Shall we just add a new parameter named like `exclude_change_log` instead of having a new method for the case?